src/AdminSettings.vue

```diff
@@ -29,85 +29,73 @@
 
 		<ul class="password-policy__settings-list">
 			<li>
-				<input id="password-policy__settings__min-length"
-					v-model="config.minLength"
-					type="number"
-					@change="updateNumberSetting('minLength')">
-				<span>{{ t('password_policy', 'Minimum password length') }}</span>
+				<label>
+					<input id="password-policy__settings__min-length"
```

Comment:
Just researched for a moment, as i didn't know the implicit label yet. I just knew the explicit version with `<label for="">`.
Thus i just wanted to mention, that i found this comment on the internet... 🤷‍♂️ 
>_Generally, explicit labels are better supported by assistive technology._
https://www.w3.org/WAI/tutorials/forms/labels/

Reply:
Fair enough :wink: